### PR TITLE
Change contact's bank account

### PR DESCRIPTION
**Pull request by chrisixion**
## 📝 What
Details on how to update a contact's bank account via the update contact endpoint.

## 🏚 Before
Screen Shot 2020-04-28 at 4 37 28 pm

## 🏡 After
Screen Shot 2020-04-28 at 4 37 18 pm

### CODE REVIEW

File: source/openapi3/split.yaml [hunk 1]
```diff
@@ -2395,12 +2395,14 @@ paths:
...
-        You can update the name and email of any Contact. This is essentially an alias you
-        can use to provide something contextually meaningful.
-
+        You can update the name, email, bank account and metadata of any Contact.
 
         <aside class="notice">
-          Any previous transactions to that Contact will retain the Contact name that was current at the time.
+          <ul>
+            <li>Previous transactions to this Contact will retain the name and bank account that was used at the time.</li>
+            <li>You cannot update a Contact's bank account details if they currently have an accepted agreement.</li>
```
Comment (astley92) [inline source/openapi3/split.yaml:2403]: I thought we allowed this and cancelled the current Agreement and replaced it with a new one? Just did a couple of tests in Sandbox and was able to do this. Perhaps remove this line or change to

Any current active Agreement with this contact will be cancelled and a new Agreement will be created in its place.

**Review by astley92 — COMMENTED** [review]
One small change suggested, all LGTM otherwise.

File: source/openapi3/split.yaml (hunk 1 above)
Comment (chrisixion) [reply source/openapi3/split.yaml:2403]: Thanks @astley92 - the docs are aimed at standard Merchants, in which case they'd have to cancel any accepted agreements first. The behaviour you're describing is for KYC trusted merchants and the documentation for that is available in Intercom.

Comment (astley92) [reply source/openapi3/split.yaml:2403]: Ahhh yep @chrisixion that makes sense.

**Review by astley92 — APPROVED** [review]

**Pull request merged**